splice_machine/src/main/java/com/splicemachine/derby/impl/storage/SpliceRegionAdmin.java

```diff
@@ -553,6 +537,20 @@ private static TableDescriptor getTableDescriptor(String schemaName, String tabl
         SpliceTransactionManager tc = (SpliceTransactionManager)lcc.getTransactionExecute();
         DataDictionary dd = lcc.getDataDictionary();
 
+        // Check parameters
+        if (schemaName == null) {
+            schemaName = lcc.getCurrentSchemaName();
+        } else {
+            schemaName = schemaName.toUpperCase();
```

Comment:
Do we handle case-sensitive schemas? 

create schema "thisIsCaseSensitive";

Reply:
We should support quoted names in our system procedure. For now, I'll keep the case as it is. If the table name is case insensitive, the user needs to specify name in upper case. I'll remove toUpperCase().

Reply:
http://www.starlink.ac.uk/star/java/db/docs/html/ref/ref-single.html: An SQL92Identifier is a dictionary object identifier that conforms to the rules of SQL-92. SQL-92 states that identifiers for dictionary objects are limited to 128 characters and are case-insensitive (unless delimited by double quotes), because they are automatically translated into uppercase by the system. 

Let's be careful here and make sure we do not change user behavior on database object names. If the schema/index is created without quotes, users will expect the name to be case-insensitive. Don't we have a central module to deal with object name searching and matching?

Reply:
We are lack of a module to normalize object names. Our behavior does not conform with SQL92 standard.

Reply:
I will keep the current behavior as it is, but we should address the following problem in a separate Jira
1) If object name is not double quoted, it is case-insensitive, convert it to upper case
2) If object name is double-quoted, remove double quotes
3) if schema name is not specified, default to current schema, instead of SPLICE.

Reply:
I see. Please create a public improvement JIRA for the proposed change. Thanks @jyuanca. Copy @genedavis too.

Reply:
Also, this is not aligned with our documentation: https://doc.splicemachine.com/sqlref_identifiers_intro.html